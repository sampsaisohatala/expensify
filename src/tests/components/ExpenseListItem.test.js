import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
import ExpenseListItem from '../../components/ExpenseListItem';
import expensesTestData from '../fixtures/expenses';

Enzyme.configure({
   adapter: new Adapter(),
});

test('should render ExpenseListItem with expense', () => {
   const wrapper = shallow(<ExpenseListItem {...expensesTestData[0]} />);
   expect(toJSON(wrapper)).toMatchSnapshot();
});
