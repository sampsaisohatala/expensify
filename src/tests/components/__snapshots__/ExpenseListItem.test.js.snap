// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseListItem with expense 1`] = `
<div>
  <Link
    to="/edit/id1"
  >
    <h3>
      Coffee
    </h3>
  </Link>
  <p>
    1.95 $
     - 
    1st of January, 1970
  </p>
</div>
`;
